_sass/components/partners/_security-experience.scss

```diff
@@ -0,0 +1,38 @@
+#security-experience-title {
```

Comment:
If there are snazzier ways to do anything in this file, let me know.

Reply:
There are several different typography classes and mixins that are used. Mostly you are adjusting line heights, so they may be overkill for your needs.